packages/plugin-ext/src/common/objects.ts

```diff
@@ -0,0 +1,233 @@
+// tslint:disable
+// copied from https://github.com/microsoft/vscode/blob/1.37.0/src/vs/base/common/objects.ts
+/*---------------------------------------------------------------------------------------------
+*  Copyright (c) Microsoft Corporation. All rights reserved.
+*  Licensed under the MIT License. See License.txt in the project root for license information.
+*--------------------------------------------------------------------------------------------*/
+
+import { isUndefinedOrNull, isArray, isObject } from './types';
```

Comment:
this file has a lot of unused code, please remove as much as possible

also check whether we already have required methods in our dependencies or in core, i.e. deepClone and deepFreeze can be found in core